packages/wrangler/package.json

```diff
@@ -98,6 +98,7 @@
 		"chokidar": "^3.5.3",
 		"esbuild": "0.14.51",
 		"miniflare": "^2.8.1",
+		"msw": "^0.46.1",
```

Comment:
Definitely should be a devDep not a dep.

Reply:
Make sure you re-run `npm i` after fixing to ensure that package-lock.json gets updated.

Reply:
Done 😄